packages/ember/tailwind3-vite/vitest.config.ts

```diff
@@ -1,5 +1,5 @@
 /// <reference types="vitest" />
-import { defineConfig } from 'vite';
+import { defineConfig } from "vite";
```

Comment:
This is strictly speaking not necessary, but any edit in that file will do this change for me, so ... why not 🤷🏾